packages/firestore/src/local/indexeddb_mutation_queue.ts

```diff
@@ -470,12 +469,9 @@ export class IndexedDbMutationQueue implements MutationQueue {
       batch
     ).next(removedDocuments => {
       this.removeCachedMutationKeys(batch.batchId);
-      if (this.garbageCollector !== null) {
-        for (const key of removedDocuments) {
-          // TODO(gsoltis): tell reference delegate that mutation was ack'd
-          this.garbageCollector.addPotentialGarbageKey(key);
-        }
-      }
+      return PersistencePromise.waitForEach(removedDocuments, key => {
+        return this.referenceDelegate.removeMutationReference(transaction, key);
```

Comment:
This line seems to be missing from Android (at least in master). Is that still coming?